coding-test/11일차/오픈채팅방.js

```diff
@@ -0,0 +1,50 @@
+function solution(record) {
+    //로직구성
+    //0. 받아온 문자열을 쪼개서 배열로 만든다.
+    //1. 객체로 새로 들어온 아이디와, 닉네임을 매치
+    //2. Enter, Leave, Change 에 따라서 닉네밍임이 변경되는 함수로 구현
+    //3. 
+    
+    const chatRoom = {};
+    const msg = [];
+    //받아온 user을 배열에 담는 함수
+    function create_user() {
```

Comment:
```suggestion
    function createUser() {
```
javascript에서는 snake_case보다는 camelCase 사용하는것이 일반적인 규칙입니다

Reply:
좋은 팁 알려주셔서 감사합니다!